pkg/util/log/log.go

```diff
@@ -46,6 +46,16 @@ const (
 	ServerErrorResultType ResultType = "InternalServerError"
 )
 
+func MapStatusCodeToResultType(statusCode int) ResultType {
```

Comment:
I think we can remove the else's{} to simplify the function (pretty common in Go).

Reply:
```go
func MapStatusCodeToResultType(statusCode int) ResultType {
	if statusCode < 300 && statusCode >= 200 {
		return SuccessResultType
	}
	if statusCode < 500 {
		return UserErrorResultType
	}
	return ServerErrorResultType
}
```